test/unit/shmem_team_ptr.c

```diff
@@ -45,7 +45,7 @@ int main(void) {
             printf("Error, this test requires more than 1 process\n");
         }
         shmem_finalize();
-        return 1;
+        return 0;
```

Comment:
I think this is not an error, rather a graceful exit. Other tests did the same, I believe. Please let me know if you find otherwise.